packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -56,40 +56,56 @@ function getProjectConfiguration(project) {
   return parsedProjectConfiguration;
 }
 
-function _getParsedClassicFilepath(pathSegments, projectConfiguration) {
-  const projectNamePathSeparators = projectConfiguration.pkg.name.split(path.sep);
-
-  pathSegments.splice(
-    0,
-    pathSegments.indexOf(projectNamePathSeparators[0]) + projectNamePathSeparators.length
-  );
-
-  return pathSegments.join(path.sep);
-}
+function _addonFound(filename, projectConfiguration) {
```

Comment:
Looking at how this is used in `getNormalizedFilePath` below, it looks like this should be named `getAddonName`, as it returns the name of the addon found in the path. This makes it make a bit more sense when it's passed as a param into the `path.join` below, as otherwise it seems like this is returning a boolean since the return value of this currently is `addonFound`.